src/agent/rustjail/src/process.rs

```diff
@@ -204,11 +206,6 @@ impl Process {
         Ok(p)
     }
 
-    pub fn notify_term_close(&mut self) {
-        let notify = self.term_exit_notifier.clone();
-        notify.notify_waiters();
```

Comment:
Hi @gaohuatao-1 

I think it's ok to only notify the term close when the process wasn't the init one, but 
can you have a try with replacing the notify.notify_waiters() with notify.notify_one() ?
I think it would be work.

Reply:
Thanks for you reply.
The function notify.notify_one() will regress https://github.com/kata-containers/kata-containers/pull/7352. 
The term_exit_notifier.notified() may be called in multiple futures, such as read_stdout() and read_stderr(), if tty of pod is false.  The first call to notified() will complete immediately, but the one after that will wait.  Pls note:  in the above scenario of tty closed, there is always one stream blocked if the exec cmd has a background process.
Of course, it will be ok if tty is true, because notified() and notify_one() are in one-to-one correspondence.